.eslintrc.json

```diff
@@ -15,39 +18,24 @@
   },
   "overrides": [
     {
-      "files": [
-        "**/*.svelte"
-      ],
-      "processor": "svelte3/svelte3",
+      "files": ["*.svelte"],
+      "parser": "svelte-eslint-parser",
       "rules": {
         "no-undef": "off"
+      },
+      "parserOptions": {
+        "parser": "@typescript-eslint/parser"
       }
     }
   ],
-  "parserOptions": {
-    "ecmaVersion": 2019,
-    "sourceType": "module"
-  },
   "rules": {
     "arrow-parens": 0,
-    "curly": [
-      2,
-      "multi-line"
-    ],
-    "quotes": [
-      2,
-      "single",
-      "avoid-escape"
-    ],
-    "semi": [
-      2,
-      "always"
-    ],
+    "curly": [2, "multi-line"],
+    "quotes": [2, "single", "avoid-escape"],
+    "semi": [2, "always"],
     "strict": 0,
     "no-use-before-define": 0,
-    "import/no-unresolved": 0
-  },
-  "settings": {
-    "svelte3/typescript": true
+    "import/no-unresolved": 0,
+    "no-inner-declarations": 0
```

Comment:
I'm guessing this one was required so there weren't a ton of errors introduced?

Reply:
There was only one error, which was the one I rolled back. I figured we could always fix it later and remove the lint rule.